### ISSUE TITLE
1.4.11 causes JEI to throw an error

### ISSUE THREAD

**Issue by jimmon89** [stfwi/wilescollection#2]
With both Forge and Neoforge, running version 1.4.11 of WilE's Collection under linux causes JEI to throw an error about an NPE
[Error Log With Neoforge](https://mclo.gs/jXAwGvH)
[Error Log With Standard Forge](https://mclo.gs/w5n8ZDY)

OS: `Garuda Linux (Arch Derivative)`
Desktop Environment: `KDE (version 5.27.10)`
Kernel Version: `6.5.12-273-tkg-bore (64-bit)`
Display Server: `X11`
Java Version: `17.0.9`
Minecraft Version: `1.20.1`
Neoforge Version: `47.1.81`
Forge Version: `47.2.8`
Last Working WilE's Collection Version: `1.4.9`
Known Broken WilE's Collection Version: `1.4.11`
JEI Version: `15.2.0.27`
Modpack: [TownCraft for 1.20.1 Version 0.85](https://www.curseforge.com/minecraft/modpacks/towncraft/files/4935753)
Launcher: `Prism Launcher`

CPU: `AMD Ryzen 5 3400G with Radeon Vega Graphics - 8 Cores`
GPU: `NVIDIA GeForce RTX 3060`
Ram: `32 GiB`

More Info Available Upon Request.
Available to test and provide feedback on experimental versions of WilE's Collection.

--EDIT--
the error message on screen is
```
java.lang.NullPointerException: Cannot invoke "String.length()" because "p_135844_" is null
```

**Comment by jimmon89** [comment]
another user in the same community is getting the same crash under windows 11, attaching their crash log
[Error Log Forge](https://mclo.gs/dWmZKxh)

**Comment by stfwi** [comment]
Hey jimmon, ty for the logs. At the first look this seems to be a load order issue. I think I can handle that in the forge mod definition file. Let me double check ; Cheer's

**Comment by jimmon89** [comment]
not a problem, happy to be of assistance

**Comment by stfwi** [comment]
Alright, could find out how to define the load order, the file is in approval on curse (https://legacy.curseforge.com/minecraft/mc-mods/wiles-collection/files/4942412). 

My problem is now: I cannot reproduce the bug in the first place, so I cannot assuringly tell if the fix actually fixes the problem. Could you take a look if the problem is fixed on your machine? I'll mark the distribution file then as release (from beta) it it's fixed. Cheer's;

**Comment by jimmon89** [comment]
sure thing, will test it in a second and report back
--Edit--
same error
[Crash Report](https://mclo.gs/OCilqG6)
[Full Log](https://mclo.gs/T6kD4m5)
just in case

**Comment by stfwi** [comment]
Hi, ty, I could also reproduce the error now after installing an additional mod. So, back to the crafting table ;)

**Comment by stfwi** [comment]
Ok, next attempt, statically derived modid. This did fix it on my test machine, but could you also quickly confirm again? Cheer's

(File in approval: https://legacy.curseforge.com/minecraft/mc-mods/wiles-collection/files/4944593)

**Comment by jimmon89** [comment]
just to confirm, the file is supposed to be version 1.4.12 correct?
[Crash Report](https://mclo.gs/0E0hn6f) with the version i just downloaded
let me know if the hashes are wrong
what i have (only need to check one, but I'm going to provide them all)
MD5: `2f50b8356f684651824d6ee59acf5e0c`
SHA1: `88ea48df68223fa0b066bee0b3ff9ed15a6496f0`
SHA256: `2ab3d6eeff0cdf6c69c100f6c384f8de1c444ad086216ae894dce0f028ae4e02`
SHA512: `8a04c1aa795ff2d2e97b353ac6df5e9f854029d6e8e60b894c7b1b336c785a5248e14a33c3f11ef89b94a752d50f185fe754dbe1ddb503bf63d589e430090ce4`

if it's the wrong file, just let me know, and i'll test again tomorrow
off to bed now
best of luck!

**Comment by stfwi** [comment]
Hi, no it's version `1.4.13`  (`wilescollection-1.20.1-neoforge-1.4.13.jar`). Strangely it's still in review on curse. I also pushed it on Modrinth (`https://modrinth.com/mod/wilescollection/version/1.4.13`). So the file is already accessible there. Cheer's ;

**Comment by jimmon89** [comment]
just tested it, and it seems to be fixed with `1.4.13`
absolute legend!
cheers! :)

**Comment by stfwi** [comment]
Oky, very cool, thanks for double checking, I'll close this one then; cheers
